runtime/doc/map.txt

```diff
@@ -1431,6 +1431,223 @@ Possible values are (second column is the short name used in listing):
     -addr=other		  ?	other kind of range
 
 
+Incremental preview ~
+                                                  *:command-preview* {nvim-api}
+It's possible to allow commands to be incrementally previewed when
```

Comment:
Active voice
```suggestion
It's possible to incrementally preview commands when
```

Reply:
I think the passive voice in this case is fine since simply using the `preview` flag isn't enough to incrementally preview the command, but it *allows* you to implement incremental preview for the command. It's not the step but rather a prerequisite, if that makes any sense

Reply:
Sounds good.